crates/apollo-compiler/src/parser.rs

```diff
@@ -245,6 +234,44 @@ impl Parser {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct MappedSource {
```

Comment:
Can this be `pub(crate)`? I’d like to minimize the public API, so after we make 1.0 "final" we’ll have fewer constraints to evolve things without yet making a 2.0 with breaking changes.